docs/blocks/pocketguide.md

```diff
@@ -0,0 +1,17 @@
+## Pocket Guide
+
+download block for the pocket guide pdf
```

Comment:
Maybe it shouldn't say "download" but "insert" or even just remove it and just say: "block for"